src/fe/inf_fe_map.C

```diff
@@ -29,6 +29,18 @@
 namespace libMesh
 {
 
+bool libmesh_isfinite(Real value)
+{
+  // check if \p value is infinite
+  if( value > std::numeric_limits<Real>::max()
+         || value <  std::numeric_limits<Real>::min() )
```

Comment:
`std::numeric_limits<Real>::min()` is a positive quantity.  For example, on my system:
```
min()=2.225074e-308
max()=1.797693e+308
epsilon()=2.220446e-16
```
so even if you want to require `c_factor` to be positive, I don't think this is a valid implementation in general.  Let me take a look at what the best way to do this is...